doc/manuals/orion-api.md

```diff
@@ -1033,6 +1033,25 @@ Note that if a custom payload is used for the notification (the field `payload`
 corresponding subscription), then a value of `custom` is used for the `Ngsiv2-AttrsFormat` header
 in the notification.
 
+An empty string value for a header key in the `headers` object will remove that header from 
```

Comment:
Question. It is the best place to this part of the section?

Reply:
By the moment it's ok. We could reorder and sub-section in a next PR, as part of the "clean and tidy" final works.

NTC